packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -91,6 +192,48 @@ export class DataProxyEngine extends Engine<DataProxyTxInfoPayload> {
   async start() {}
   async stop() {}
 
+  private propagateResponseExtensions(extensions: DataProxyExtensions): void {
+    const tracingConfig = getTracingConfig(this.config.previewFeatures || [])
+
+    if (extensions?.logs?.length) {
+      extensions.logs.forEach((log) => {
+        switch (log.level) {
+          case 'debug':
+          case 'error':
+          case 'trace':
+          case 'warn':
+          case 'info':
+            // TODO these are propgated into the response.errors key
+            break
+          case 'query': {
+            let dbQuery =
+              typeof log.attributes?.query === 'string' && log.attributes?.query ? log.attributes.query : log.name
```

Comment:
```suggestion
              typeof log.attributes?.query === 'string' ? log.attributes.query : log.name
```

I think the second condition is already implied by the first one

Reply:
Yes, it is, it means attributes is not undefined. Addressing it.